File: Python/Program/DPR.py
class DPR: #Class DPR
    #Private Attributes
    __id = ""#Id
    __name = ""#Nama
    __nameBidang = ""#Nama Bidang
    __namePartai = ""#Nama Partai
    def __init__(self, id = "", name = "", nameBidang = "", namePartai = ""):#Constructor with parameter
        self.__id = id
        self.__name = name
        self.__nameBidang = nameBidang
        self.__namePartai = namePartai
    #Gette-Setter
    def getId(self):
        return self.__id

    def setId(self, id):
        self.__id = id

    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

    def getNameBidang(self):
        return self.__nameBidang

    def setNameBidang(self, nameBidang):
        self.__nameBidang = nameBidang

    def getNamePartai(self):
        return self.__namePartai

    def setNamePartai(self, namePartai):
        self.__namePartai = namePartai

File: PHP/Program/DPR.php
<?php

class DPR//Class DPR
{   //Private Attributes
    private $id = '';//ID
    private $name = '';//Nama
    private $nameBidang = '';//Nama bidang
    private $namePartai = '';//Nama partai
    private $foto = '';//Foto anggota
    //Public Method
    public function __construct($id = '', $name = '', $nameBidang = '', $namePartai = '', $foto = '')//Constructor with parameters
    {
        $this->id = $id;
        $this->name = $name;
        $this->nameBidang = $nameBidang;
        $this->namePartai = $namePartai;
        $this->foto = $foto;
    }
    //Getter-Setter
    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getName()
    {
        return $this->name;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function getNameBidang()
    {
        return $this->nameBidang;
    }

    public function setNameBidang($nameBidang)
    {
        $this->nameBidang = $nameBidang;
    }

    public function getNamePartai()
    {
        return $this->namePartai;
    }

    public function setNamePartai($namePartai)
    {
        $this->namePartai = $namePartai;
    }

    public function getFoto()
    {
        return $this->foto;
    }

    public function setFoto($foto)
    {
        $this->foto = $foto;
    }
}

?>
